static/js/containers/UserPage_test.js

```diff
@@ -77,13 +78,64 @@ describe("UserPage", function() {
           getElementsByClassName('delete-button')[0];
       };
 
+      beforeEach(() => {
+        let userProfile = Object.assign({}, USER_PROFILE_RESPONSE, {
+          username: SETTINGS.username
+        });
+        userProfile.education.push({
+          "id": 3,
+          "degree_name": DOCTORATE,
+          "graduation_date": "2015-12-01",
+          "field_of_study": "Philosophy",
+          "school_name": "Harvard",
+          "school_city": "Cambridge",
+          "school_state_or_territory": "US-MA",
+          "school_country": "US",
+          "online_degree": false
+        });
+        helper.profileGetStub.
+          withArgs(SETTINGS.username).
+          returns(Promise.resolve(userProfile));
+      });
+
       it('shows the education component', () => {
         return renderComponent(`/users/${SETTINGS.username}`, userActions).then(([, div]) => {
           let title = div.getElementsByClassName('profile-card-title')[1];
           assert.equal(title.textContent, 'Education');
         });
       });
 
+      it('should show the entries in resume order', () => {
```

Comment:
yeah, we can probably abstract some of this - I tend not to do that for stuff like this because I tend to think over-abstraction in testing can make tests much harder to read.

I'll add some more comments to explain what we're testing more.